src/main/java/com/amazon/ion/impl/bin/IonManagedWriter_1_1.kt

```diff
@@ -266,6 +268,8 @@ internal class IonManagedWriter_1_1(
         macrosById.clear()
         macroTable.clear()
         newMacros.clear()
+
+        needsIVM = true
```

Comment:
There's a `TODO` comment earlier in the method body that is resolved by this change. Can you remove that comment, please?

Reply:
Done